charts/karavi-observability/templates/karavi-topology.yaml

```diff
@@ -46,18 +46,24 @@ spec:
               path: localhost.crt
             - key: tls.key
               path: localhost.key
+        - name: karavi-topology-configmap
+          configMap:
+            name: karavi-topology-configmap
       serviceAccount: {{ .Release.Name }}-topology-controller
       containers:
       - name: karavi-topology
         image: {{ .Values.karaviTopology.image }}
         resources: {}
+        envFrom:
```

Comment:
If we are mounting the ConfigMap as a volume, we may not require this `envFrom` block which would also set them as environment variables.

Reply:
fixed